<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::East</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="Explorer" name="explorer_tab">
       <attribute name="title">
        <string>Explorer</string>
       </attribute>
      </widget>
      <widget class="Dashboard" name="dahboard_tab">
       <attribute name="title">
        <string>Dashboard</string>
       </attribute>
      </widget>
      <widget class="Simulator" name="simulator_tab">
       <attribute name="title">
        <string>Simulator</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Explorer</string>
    </property>
    <addaction name="actionTopics"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPublish"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDashboard">
    <property name="title">
     <string>Dashboard</string>
    </property>
    <addaction name="actionLoadDashboard"/>
    <addaction name="separator"/>
    <addaction name="actionSaveDashboard"/>
    <addaction name="actionSaveDashboard_As"/>
   </widget>
   <widget class="QMenu" name="menuSimulator">
    <property name="title">
     <string>Simulator</string>
    </property>
    <addaction name="actionLoadSimulator"/>
    <addaction name="separator"/>
    <addaction name="actionSaveSimulator"/>
    <addaction name="actionSaveSimulator_As"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDashboard"/>
   <addaction name="menuSimulator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTopics">
   <property name="text">
    <string>Topics</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionPublish">
   <property name="text">
    <string>Publish</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSaveDashboard">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveDashboard_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionLoadDashboard">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionLoadSimulator">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSaveSimulator">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveSimulator_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Explorer</class>
   <extends>QWidget</extends>
   <header>explorer.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Dashboard</class>
   <extends>QWidget</extends>
   <header>dashboard.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Simulator</class>
   <extends>QWidget</extends>
   <header>simulator.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
